sources/packages/web/src/services/AuthService.ts

```diff
@@ -1,39 +1,47 @@
-import Keycloak from "keycloak-js"
-import store from "../store/index"
+import Keycloak from "keycloak-js";
+import store from "../store/index";
+import { AppConfig } from "./AppConfigService";
 
-const keycloak = Keycloak({
-  url: "https://dev.oidc.gov.bc.ca/auth/",
-  realm: "jxoe2o46",
-  clientId: "student"
-});
+let keycloak: Keycloak.KeycloakInstance;
+export default async function(config: AppConfig): Promise<Keycloak.KeycloakInstance> {
+  if (keycloak) {
+    return keycloak;
+  }
+  keycloak = Keycloak({
+    url: config.authConfig.url,
+    realm: config.authConfig.realm,
+    clientId: config.authConfig.clientId
+  });
 
-keycloak.init({ onLoad: "check-sso", responseMode: "query", checkLoginIframe: false })
-    .then(() => {
-        if(keycloak.authenticated){
-            store.dispatch("auth/login", keycloak);
-        }
-        else {
-            store.dispatch("auth/logout");
-        }
+  try {
+      await keycloak.init({
+        onLoad: "check-sso",
+        responseMode: "query",
+        checkLoginIframe: false
+      });
+      if (keycloak.authenticated) {
+        store.dispatch("auth/login", keycloak);
+      } else {
+        store.dispatch("auth/logout");
+      }
 
-        if(keycloak && keycloak.idTokenParsed){
-            if(keycloak.idTokenParsed.iat){
-                const iatMiliseconds = keycloak.idTokenParsed.iat * 1000;
-                console.log("Issue Date (iat): " + new Date(iatMiliseconds));
-            }
+      if (keycloak && keycloak.idTokenParsed) {
```

Comment:
I believe that we can get rid of these debug codes now.

Reply:
Fixed